### DISCUSSION TITLE
Is there any option to add "notes" or "sources" to markdown tables using quarto?

### DISCUSSION THREAD

**Discussion by friosavi** [quarto-dev/quarto-cli#6079] [Q&A]
### Description

Im currently working on a small report, for which I'm trying to create a dynamic document using Quarto.
As of right now, im creating most tables using markdown tables.
Now, I know how to add Title's and reference to them. However, I am not sure how to add Notes to those tables.

For example, the following code would produce the table I want, with the title

```qmd
| Col1 | Col2 | Col3 |
|------|------|------|
| A    | B    | C    |
| E    | F    | G    |
| A    | G    | G    |
: First Table {#tbl-first}
```

No what If i want to add a source: "Source: Me(2023)" or other comments of a similar tone? 

Is there any way to do that?

Thank you!
F

**Comment by mcanouil** [comment]
Not really, especially using pipe tables.

For complex tables, you should consider using HTML tables features that comes with Quarto 1.3 and later version, see <https://quarto.org/docs/authoring/tables.html#html-tables>.

**Comment by friosavi** [comment]
Thank you for the suggestion.
Im currently finding it harder to use HTML tables. Since Im working with Stata and nbstata to make the document, I couldn't find an easy way to import html tables produced by `esttab` and `estout` (User written commands).

I think I can do without for now. Perhaps at some point some options are provided!

Thank you

**Reply by mcanouil** [reply]
You can also look for Grid Tables instead of pipe tables, see <https://pandoc.org/MANUAL.html#tables>.
